### PR TITLE
patch OrderPOCreate process, see issue #1649

### PR DESCRIPTION
**Pull request by homebeaver**
Hi @yamelsenih, this is the pull request as promised today.
You can test it.

A remark on BOM products. I explode the BOM but I did not implement the purchasing of components of the BOM-product. 

regards

### CODE REVIEW

File: base/src/org/compiere/process/OrderPOCreate.java [hunk 1]
```diff
@@ -53,6 +59,36 @@
 	/** Drop Ship			*/
 	private boolean		p_IsDropShip = false;
 	
+	private int m_explosion_level = 0;
```
Comment (yamelsenih) [inline base/src/org/compiere/process/OrderPOCreate.java]: Is nice add comments to new methods

**Comment by homebeaver** [comment]
Hi @yamelsenih ,
I've accidently merged develop to my branch and got a huge number of commits.
No Idea how to unmake it :-(
Only the last commit is what you wanted "subclass from *Abstract".
regards

**Comment by e-Evolution** [comment]
@homebeaver please close this pull request and send again a new only with change and based in branch where only have the changes to solve this issue.

**Comment by homebeaver** [comment]
close it - all necessary changes are now in #1866

**Pull request closed without merging**